### PR TITLE
chore(docs): Table of Contents & Code of Conduct

### PR DESCRIPTION
**Pull request by 1nb0und**
Changes

- add table of contents
- update code of conduct and contact info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -526,4 +547,4 @@ And then access them via http://localhost:8080/actuator/metrics/.
 
 # Code of Conduct
 
-This project adheres to the Contributor Covenant [Code of Conduct](/.github/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to code-of-conduct@zeebe.io.
+This project adheres to the Contributor Covenant [Code of Conduct](/.github/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please [report unacceptable behavior](https://camunda.com/events/code-conduct/reporting-violations/) as soon as possible.
```
Comment (jwulf) [inline README.md:550]: This should probably have the updated title (Community Code of Conduct).

**Review by berndruecker — APPROVED** [review]
Not a strong opinion on this myself

**Comment by akeller** [comment]
Bring in @xomiamoore, DRI for Camunda Community Hub for CoC changes. They should have a consistent template or link to the right content.

**Comment by xomiamoore** [comment]
Thanks for the ping @1nb0und -- the way we handle our CoC for the Community Hub itself is by linking to the [Community Code of Conduct page on Camunda's website](https://camunda.com/events/code-conduct/), which will always have the most up-to-date version. For an example, see [this page](https://github.com/camunda-community-hub/community/blob/main/CODE_OF_CONDUCT.MD). 

You could also add that link to CONTRIBUTING.md instead of having a full page for the CoC. :)

**Pull request merged**